components/brave_account/brave_account_service.cc

```diff
@@ -307,13 +313,15 @@ void BraveAccountService::VerifyResult() {
   request.wait = false;
   request.headers.SetHeader("Authorization",
                             base::StrCat({"Bearer ", verification_token}));
-  Client<endpoints::VerifyResult>::Send(
-      url_loader_factory_, std::move(request),
-      base::BindOnce(&BraveAccountService::OnVerifyResult,
-                     weak_factory_.GetWeakPtr()));
+  previous_request =
+      Client<endpoints::VerifyResult>::Send<RequestCancelability::kCancelable>(
```

Comment:
If you choose to refrain from adding a class member for the current request, I would recommend renaming this variable for readability reasons.

Reply:
https://github.com/brave/brave-core/pull/32155/commits/a35f63702d517be920364b77169457b3e757b614